src/AutoRest.CSharp/Common/Output/Builders/XmlSerializationMethodsBuilder.cs

```diff
@@ -457,7 +457,7 @@ private static ValueExpression DeserializeValue(XmlValueSerialization serializat
 
                     if (frameworkType == typeof(object))
                     {
-                        return element.GetObjectValue(serialization.Format.ToFormatSpecifier());
+                        return element.Value;
```

Comment:
in the extension, the `XElement.GetObjectValue` method is just calling `element.Value`, therefore we just remove it to call this directly